### PR TITLE
Convert clasp create to inquirer functionality

### PR DESCRIPTION
**Pull request by campionfellin**
Now, if you don't specify a script name, we use inquirer to ask for one.

Also added tests for this.

Signed-off-by: campionfellin <campionfellin@gmail.com>

Addresses https://github.com/google/clasp/issues/57

- [x] `npm run test` succeeds.
- [ ] Appropriate changes to README are included in PR.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -606,7 +624,7 @@ commander
             logError(error, ERROR.CREATE);
           });
         });
-      });
+      });
```
Comment (campionfellin) [inline index.ts]: Looks like we should also add no-trailing-whitespace (https://palantir.github.io/tslint/rules/no-trailing-whitespace/) to the tslint

File: index.ts [hunk 2]
```diff
@@ -209,6 +210,16 @@ Did you provide the correct scriptId?`,
   UNAUTHENTICATED: 'Error: Unauthenticated request: Please try again.',
 };
 
+// Questions (prompts) for clasp create
+const createQuestions = [
+  {
```
Comment (grant) [inline index.ts]: `[{`

File: index.ts [hunk 3]
```diff
@@ -209,6 +210,16 @@ Did you provide the correct scriptId?`,
   UNAUTHENTICATED: 'Error: Unauthenticated request: Please try again.',
 };
 
+// Questions (prompts) for clasp create
+const createQuestions = [
```
Comment (grant) [inline index.ts]: inline

File: index.ts [hunk 4]
```diff
@@ -209,6 +210,16 @@ Did you provide the correct scriptId?`,
...
+  {
+    type : 'input',
+    name : 'title',
+    message : 'give a script title: ',
+    default: LOG.UNTITLED_SCRIPT_TITLE
```
Comment (grant) [inline index.ts]: Add trailing comma.

File: index.ts (hunk 1 above)
Comment (grant) [reply index.ts]: 👍

**Pull request merged**